dart/lib/src/protocol/contexts.dart

```diff
@@ -25,6 +25,32 @@ class Contexts extends MapView<String, dynamic> {
           Gpu.type: gpu,
         });
 
+  factory Contexts.fromJson(Map<String, dynamic> data) => Contexts(
```

Comment:
if `data` has extra keys that are not "typed" by Contexts, they will be lost, we need to add all the fields.

Reply:
✅ good catch, thx